gql_websocket_link/lib/src/link.dart

```diff
@@ -1,66 +1,208 @@
 import "dart:async";
+import "dart:convert";
 
 import "package:gql_exec/gql_exec.dart";
 import "package:gql_link/gql_link.dart";
-import "package:gql_websocket_link/src/get_from_ast.dart";
-import "package:gql_websocket_link/src/socket_client.dart";
 import "package:gql_websocket_link/src/messages.dart";
-import "package:gql/language.dart";
-import "package:gql/ast.dart";
+import "package:rxdart/rxdart.dart";
+import "package:uuid_enhanced/uuid.dart";
+import "package:web_socket_channel/io.dart";
```

Comment:
Isn't `WebSocketChannel` what we want for portability?
```suggestion
import "package:web_socket_channel/web_socket_channel.dart";
```